libraries/ms-common/test/test_gpio.c

```diff
@@ -2,13 +2,18 @@
 
 #include "extra_unity.h"
 #include "gpio.h"
+#include "gpio_cfg.h"
 #include "unity.h"
 
 void setup_test(void) { }
 
 void teardown_test(void) { }
 
-#define INVALID_PORT_PIN 100
+#define VALID_PIN 0
+#define VALID_PORT 0
```

Comment:
How are you planning on handling cases where ports start with 1? (i.e. the MSP430)